ts-packages/web/src/app/(social)/_components/create-repost.tsx

```
@@ -0,0 +1,556 @@
+              <div className="flex items-center gap-2">
+                <span className="text-white font-medium text-lg">
+                  {authorName || 'Anonymous'}
+                </span>
+              </div>
+              <Certified className="size-5" />
+            </div>
+
+            {feedcontent && (
+              <div
+                className="prose prose-invert text-sm p-3 bg-neutral-800 text-neutral-300 mb-3 font-light"
+                dangerouslySetInnerHTML={{ __html: feedcontent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)